heft-plugins/heft-serverless-stack-plugin/src/ServerlessStackPlugin.ts

```diff
@@ -16,15 +16,16 @@ import type {
 import { FileSystem, Import, SubprocessTerminator } from '@rushstack/node-core-library';
 import type {
   PluginName as Webpack4PluginName,
-  IWebpackPluginAccessor as IWebpack4PluginAccessor
+  IWebpack4PluginAccessor
```

Comment:
I think I liked how this was before better. It doesn't make sense for the APIs to contain the major version when that's already in the package name.